src/main/java/bio/terra/controller/RepositoryApiController.java

```diff
@@ -70,6 +70,7 @@
     private final IngestRequestValidator ingestRequestValidator;
     private final FileService fileService;
     private final PolicyMemberValidator policyMemberValidator;
+    private final AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
 
     // needed for local testing w/o proxy
     private final ApplicationConfiguration appConfig;
```

Comment:
can this be removed since the local auth user factory autowires it directly?

Reply:
don't I need it for other methods eg: datasetService.createDataset(datasetRequest, getAuthenticatedInfo()) ....or should I be factoring it out there as well?